src/extension/background-script/actions/lnurl/index.ts

```diff
@@ -20,10 +21,11 @@ async function lnurl(message: Message) {
 
   switch (lnurlDetails.tag) {
     case "channelRequest":
-      console.error("lnurl-channel is not implemented");
-      // TODO: add support for LNURL channel
-      // connectPeer method in connector
-      return { error: "not implemented" };
+      if ("public" in message) {
```

Comment:
what is this check about? and why only a `return` in the else case?

Reply:
hmm. now we got another else there. and all the other lnurl options are missing. 
should that then be 

```js 
lnurl(MessageLNURLChannel | MessageLNURLPay | MessageLNURLAuth | ...)
```

and where is that `public` coming from?